### PR TITLE
refactor: ui support navigator

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/features/auth/hooks/useMetamask.ts [hunk 1]
```diff
@@ -15,6 +16,11 @@ const useMetamask = () => {
   const accountsChanged = async (newAccount: string) => setAccount(newAccount);
 
   const connect = async () => {
+    if (!navigator.userAgent.toLowerCase().match(/chrome|firefox|edge|brave/)) {
+      dispatch(openToast({ text: 'Navigator is not compatible with MetaMask', type: 'error' }));
```
Comment (mmiranda-stackit) [inline src/features/auth/hooks/useMetamask.ts:20]: In my opinion, 'Browser' is a more standard way to name web browsers/navigators

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]
There is a small suggestion on naming. Also, I do believe that we need to revisit the eslint rules, it looks like we are abusing of inline disabling rules. I think that we need to refine our global eslint file so we don't end up with ton of warning in our code base.

File: src/features/auth/hooks/useMetamask.ts (hunk 1 above)
Comment (AlefrankM) [reply src/features/auth/hooks/useMetamask.ts]: This change were applied

**Review by america-nishi — APPROVED** [review]
looks good

**Pull request closed without merging**